src/components/shared/CallNodeContextMenu.js

```diff
@@ -447,151 +449,210 @@ class CallNodeContextMenuImpl extends React.PureComponent<Props> {
 
     return (
       <>
-        <TransformMenuItem
-          shortcut="m"
-          icon="Merge"
-          onClick={this._handleClick}
-          transform="merge-function"
-          title={oneLine`
-            Merging a function removes it from the profile, and assigns its time to the
-            function that called it. This happens anywhere the function was called in
-            the tree.
-          `}
+        <Localized
```

Comment:
These diffs are not really fun with spacing changes. It could be easier to review with checking the "hide whitespace changes"